### DISCUSSION TITLE
Highlighting for inline maths delimiter `\(...\)`

### DISCUSSION THREAD

**Discussion by harrysw1729** [James-Yu/LaTeX-Workshop#3677] [Q&A]
Does the extension support pair highlighting for the inline maths delimiter `\(...\)` similar to other bracket pair highlighting such as `\{...\}`? I could not get any highlighting for the inline maths delimiter, but there appeared to be a past commit in 2018 (bd5ff304483d4470fd84f0b38937da8bd18e1bc8 ) that supposedly supported pair highlighting for `\(...\)`.

Thank you for your help.

**Comment by jlelong** [comment]
Thanks for the issue. It will be fixed shortly in https://github.com/jlelong/vscode-latex-basics/issues/42
